README.md

```diff
@@ -3,24 +3,23 @@
 [![Build Status](https://travis-ci.org/JuliaLang/JuliaParser.jl.svg?branch=master)](https://travis-ci.org/JuliaLang/JuliaParser.jl?branch=master)
 [![Coverage Status](https://img.shields.io/coveralls/jakebolewski/JuliaParser.jl.svg)](https://coveralls.io/r/jakebolewski/JuliaParser.jl)
 
-A pure Julia port of Julia's parser.  It should be fully compatible with Julia's built in parser and it correctly parses all ~3000+ Julia source files currently available in Julia's 300+ user packages and all Julia source code in Base.
+A pure Julia port of [Julia](https://github.com/JuliaLang/julia)'s [parser](https://github.com/JuliaLang/julia/blob/master/src/julia-parser.scm).  It should be fully compatible with Julia's built in parser and it correctly parses all ~3000+ Julia source files currently available in Julia's [300+ user packages](http://pkg.julialang.org/) and all Julia source code in [Base](https://github.com/JuliaLang/julia/tree/master/base).
 
 Differences with Julia's Flisp Parser
 -------------------------------------
-* BigInt and Int128 numbers are treated as literal values instead of expressions.
-* Literal Negation is done as negated literals rather than using Expr(:-)
-* QuoteNode's are replaced with Expr(:quote).
+* `BigInt` and `Int128` numbers are treated as literal values instead of expressions.
+* Literal negation is done as negated literals rather than using `Expr(:-)`
+* `QuoteNode`s are replaced with `Expr(:quote)`.
 
 Using JuliaParser as your primary parser
 -------------------------------------
-JuliaParser provides a script that will replace the builtin flips parser by itself.
+JuliaParser provides a script that will replace the builtin [flisp](https://github.com/JeffBezanson/femtolisp) parser by itself.
```

Comment:
I want to avoid giving the impression that people need to understand what flisp is, since other than being the language that the original parser was written in, it's not really relevant here. If people come across the original parser and find it's source code, they will find out about flisp.